.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Is that correct?

Reply:
This is auto-generated by husky and ignores `husky.sh` within the directory `_`, so seems correct for me 🤷